java/maven/src/org/netbeans/modules/maven/execute/CommandLineOutputHandler.java

```diff
@@ -390,7 +422,7 @@ private ExecutionEventObject parseExecEvent(String line) {
          * or is another text, and update folds accordingly.
          */
         private void updateFoldForException(String line) {
-            if (stackTraceElement.matcher(line).find()) {
+            if (line.endsWith(")") && stackTraceElement.matcher(line).matches()) {
```

Comment:
now I remember why i switched it to find() in the first place: the folding code was already using `find()` :)

updated the regexp and switched everything to `matches()` + added a fast path for even less overhead.